playbooks/075-update.yml

```diff
@@ -0,0 +1,21 @@
+- name: Run pre_update hooks
+  vars:
+    hooks: "{{ pre_update | default([]) }}"
+    step: pre_update
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- name: Update playbook
```

Comment:
I followed convention used in other playbooks. I don't think we need any rescue block, because there is no rollback operation yet. We might change it in future, but for now current structure is enough.